pkg/ebpf/probes/probes.go

```diff
@@ -7,26 +7,28 @@ import (
 	bpf "github.com/aquasecurity/libbpfgo"
 )
 
+// Probes
+
 type Probes interface {
+	// attach
 	Attach(probe Handle) error
+	AttachNetProbes() error
```

Comment:
Can't we handle the net probes the same way we handle probes and just add them as event dependencies? Then we would load them using the same logic we do for standard probes. It would also help streamline the net events code in tracee.go